app/src/main/java/com/concordium/wallet/ui/cis2/lookfornew/TokenSelectedDestination.kt

```diff
@@ -0,0 +1,8 @@
+package com.concordium.wallet.ui.cis2.lookfornew
+
+enum class TokenSelectedDestination {
+    TOKEN,
+    NFT,
+    MIXED,
```

Comment:
It seems that if both types are added, we are always navigated back to fungibles.